CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 # Changelog
+## 5.1.0 (June 7, 2018)
+
+- Add Scatter Plot ([commit](https://github.com/gooddata/gooddata-react-components/commit/77b748a40b620a5233a370bf8110fdba87b282f3))
+- Add Donut Chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/a703cd78f5095c6df6c73c716c7865b17913ad73))
+- Finish moving of code from @gooddata/indigo-visualization react-components, making indigo-visualizations deprecated ([commit](https://github.com/gooddata/gooddata-react-components/commit/7a5ec756f39d4d222e2eb102f94f929aae0d44fc))
+- Increase TypeScript usage ([commit](https://github.com/gooddata/gooddata-react-components/commit/265fda101384f83fb2b530192c0d065de2970ba2))
+- Extend existing and add new examples
+- And many more bugfixes and minor improvements
+
```

Comment:
- Added Scatter Plot chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/77b748a40b620a5233a370bf8110fdba87b282f3))
- Added Donut chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/a703cd78f5095c6df6c73c716c7865b17913ad73))
- Moved the code from @gooddata/indigo-visualization react-components and deprecated indigo-visualizations ([commit](https://github.com/gooddata/gooddata-react-components/commit/7a5ec756f39d4d222e2eb102f94f929aae0d44fc))
- Increased the usage of TypeScript ([commit](https://github.com/gooddata/gooddata-react-components/commit/265fda101384f83fb2b530192c0d065de2970ba2))
- Extended the existing examples and added new ones
- Added miscellaneous bugfixes and minor improvements